@here/harp-mapview/lib/MapView.ts

```diff
@@ -767,6 +767,7 @@ export class MapView extends THREE.EventDispatcher {
     private m_thisFrameTilesChanged: boolean | undefined;
     private m_lastTileIds: string = "";
     private m_languages: string[] | undefined;
+    private m_politicalView: string | null;
```

Comment:
The problem with this is the functionality of ITileDecoder->configure() method, which does not override (re-write) options if they are undefined, thus setting default Political Point of View (which would be undefined) is not possible on decoder, because it will be simply ignored.
We may have two solutions on that:
- extend ITileDecoder->configure() which additional parameter for political view (the same as it's done with languages, but in my opinion breaks the interface, because political view is part of options - passed via OptionsMap).
- make a contract that empty string "" for point of view resets to default behavior.

Reply:
we do not use "null" internally, we use "undefined" only. There must be a way to keep this principle.